init.go

```diff
@@ -70,7 +75,15 @@ func LogInit() {
 func SetDebug(enabled bool) {
 	if enabled {
 		debugFlag = true
-		Debug = log.New(os.Stdout, "debug:", log.Ldate|log.Ltime|log.Lshortfile)
+		if filename := os.Getenv("DEBUG_LOG"); filename != "" {
+			file, err := os.OpenFile(filename, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```

Comment:
can we use `reopen()`, update it to not call `dup2()` if fd is not passed.

Reply:
yes, i had make the changes as requested.